crawl/workspace.py

```diff
@@ -170,19 +160,25 @@ def _parse_maven_install(self, maven_install_info, repo_root_path):
                 transitives_list.append({dep : transitives})
                 self.dependency_metadata.register_exclusions(dep, exclusions)
 
-        # Overrides the deps honoring the override file
-        for key, dep in result.items():
+        # Overrides the direct deps mapping
+        overridden_result = copy.deepcopy(result)
```

Comment:
I don't think you need deepcopy here? You only want to copy the list, not the dependencies in the list?  I think it would be more readable to start with an empty dict and then in the loop below populating it, instead of copying it first and then overriding some keys.